### PR TITLE
Enable/disable vdc

### PR DESCRIPTION
**Pull request by jvwilder**
#### What this PR resolves:

#86: enable/disable vdc + tests/docs

### CODE REVIEW

**Comment by jvwilder** [comment]
Requires Jumpscale/lib9#46

File: templates/ovc/vdc/actions.py [hunk 1]
```diff
@@ -161,10 +169,44 @@ def processChange(job):
...
-        raise j.exceptions.AYSNotFound("no producer g8client found. cannot continue init of %s" % service)
+        raise j.exceptions.AYSNotFound("No producer g8client found. Cannot continue uninstall of %s" % service)
 
     g8client = service.producers["g8client"][0]
     cl = j.clients.openvcloud.getFromService(g8client)
     acc = cl.account_get(service.model.data.account)
     space = acc.space_get(service.model.dbobj.name, service.model.data.location)
     space.delete()
+
+def enable(job):
```
Comment (ahussein) [inline templates/ovc/vdc/actions.py]: Can you please add docstring ?

File: templates/ovc/vdc/actions.py [hunk 2]
```diff
@@ -161,10 +169,44 @@ def processChange(job):
...
+    acc = cl.account_get(service.model.data.account)
+    # Get space, raise error if not found
+    space = acc.space_get(name=service.model.dbobj.name,
+                          location=service.model.data.location,
+                          create=False)
+    space.enable('The space should be enabled.')
+    service.model.data.disabled = False
+    service.save()
+
+def disable(job):
```
Comment (ahussein) [inline templates/ovc/vdc/actions.py]: Can you please add docstring ?

File: tests/bp_test_templates/basic/vdc_ovc/003_test_enable_vdc.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+g8client__env:
+    url: 'du-conv-2.demo.greenitglobe.com'
+    login: 'hamdya'
+    password: 'Abdoholmez.1'
+    account: 'hamdya'
```
Comment (ahussein) [inline tests/bp_test_templates/basic/vdc_ovc/003_test_enable_vdc.yaml]: can you please remove the actual user info from the test bp and use placeholders instead ?

File: templates/ovc/vdc/actions.py [hunk 4]
```diff
@@ -161,10 +169,44 @@ def processChange(job):
...
+    g8client = service.producers["g8client"][0]
+    cl = j.clients.openvcloud.getFromService(g8client)
+    acc = cl.account_get(service.model.data.account)
+    # Get space, raise error if not found
+    space = acc.space_get(name=service.model.dbobj.name,
+                          location=service.model.data.location,
+                          create=False)
+    space.enable('The space should be enabled.')
+    service.model.data.disabled = False
+    service.save()
```
Comment (alichaddad) [inline templates/ovc/vdc/actions.py]: It would be better if you used `service.saveAll()`

File: templates/ovc/vdc/actions.py [hunk 5]
```diff
@@ -161,10 +169,44 @@ def processChange(job):
...
+    g8client = service.producers["g8client"][0]
+    cl = j.clients.openvcloud.getFromService(g8client)
+    acc = cl.account_get(service.model.data.account)
+    # Get space, raise error if not found
+    space = acc.space_get(name=service.model.dbobj.name,
+                          location=service.model.data.location,
+                          create=False)
+    space.disable('The space should be disabled.')
+    service.model.data.disabled = True
+    service.save()
```
Comment (alichaddad) [inline templates/ovc/vdc/actions.py]: Same as previous comment use `saveAll`

**Pull request merged**